website/docs/icons/usage-guidelines/partials/guidelines/overview.md

```diff
@@ -119,8 +119,21 @@ Some icons represent common actions within our products.
 
 ### Help actions
 
+- Use `docs-link` when linking to documentation pages and installation guides.
+
+- Use `learn-link` when linking to tutorials.
+
+- Use `support` when referencing HashiCorp support.
+
 ![Examples of common help icons](/assets/foundations/flight-icons/help-actions.png =660x*)
 
+!!! Info
+
+**Other actions**
+
+Use `external-link` for any other help action that points to an external URL.
+!!!
+
```

Comment:
Curious...why add this banner? The `external-link` is documented in "Navigation actions" directly above this section, so it feels unnecessarily repetitive and prominent.